src/types/options.ts

```diff
@@ -261,6 +261,12 @@ Use it along with "enableTime" to create a time picker. */
   /* Displays time picker in 24 hour mode without AM/PM selection when enabled.*/
   time_24hr: boolean;
 
+  /* Update input value attribute when a date has been selected,
+  - Used in conjunction with altInput config since this will prevent the update value of the hidden input.
+  - With this approach issues related with different assigned values to this input will be fixed since it must be 
+  Controlled by developer at the OnSelectedDate callback */
+  updateInputVal: boolean;
```

Comment:
I am little againts adding more and more IMHO quite unneded options. Can the solution be made without the additional option?

Anyway, please add a test to reproduce the initial problem and assert it is fixed.